operator/internal/lifecycle/constants.go

```diff
@@ -17,6 +17,7 @@ const (
 	defaultOperatorLabel  = "cluster.redpanda.com/operator"
 	defaultOwnerLabel     = "cluster.redpanda.com/owner"
 	generationLabel       = "cluster.redpanda.com/generation"
+	configVersionLabel    = "cluster.redpanda.com/configVersion"
```

Comment:
Can we make this more specific? Config could mean anything. Is this a node config, cluster config, or a more general concept of config?

Reply:
This is a reimplementation of a cluster configuration version annotation for rolling pods, but with labels as we're using labels elsewhere too.